File: crackme.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("password: ", function(password) {
    if (password === "crackme") {
        console.log("success!");
    } else console.log("fail.");

    rl.close();
});

File: crackme.go
package main

import (
    "fmt"
)

func main() {
    var password string
    fmt.Println("password: ")
    fmt.Scan(&password)
    if password == "crackme" {
        fmt.Println("success!")
    } else {
        fmt.Println("fail.")
    }
}